eng/pipelines/libraries/build-job.yml

```diff
@@ -50,22 +50,21 @@ jobs:
       ${{ if ne(parameters.liveRuntimeBuildConfig, '') }}:
         dependsOn:
         - ${{ if ne(parameters.osGroup, 'WebAssembly') }}:
-          # Use corelib dependencies for coreclr and non test builds
-          - ${{ if and(eq(parameters.runtimeFlavor, 'coreclr'), eq(parameters.testScope, '')) }}:
-            - ${{ format('coreclr_corelib_build_{0}{1}_{2}_{3}', parameters.osGroup, parameters.osSubgroup, parameters.archType, parameters.liveRuntimeBuildConfig) }}
+          # Use full product dependency for non-coreclr and test builds
           - ${{ if or(ne(parameters.runtimeFlavor, 'coreclr'), ne(parameters.testScope, '')) }}:
             - ${{ format('{0}_product_build_{1}{2}_{3}_{4}', parameters.runtimeFlavor, parameters.osGroup, parameters.osSubgroup, parameters.archType, parameters.liveRuntimeBuildConfig) }}
-        - ${{ if eq(parameters.osGroup, 'WebAssembly') }}:
-          - ${{ format('coreclr_corelib_build_{1}{2}_{3}_{4}', parameters.runtimeFlavor, 'linux', parameters.osSubgroup, 'x64', parameters.liveRuntimeBuildConfig) }}
 
       variables:
         - _subset: libs
         - _addtionalBuildArguments: ''
```

Comment:
Nit: Now that you're touching this if you're willing to fix it, this has a typo. `addtionalBuildArguments` -> `additionalBuildArguments`.

Reply:
done